snuba/web/db_query.py

```diff
@@ -503,9 +502,6 @@ def _raw_query(
         elif isinstance(cause, ExecutionTimeoutError):
             status = QueryStatus.TIMEOUT
 
-        if request_status.slo == SLO.AGAINST:
```

Comment:
We raise a `QueryException` later and I think we just end up double reporting to Sentry - if there is a reason to keep this line lemme know